### DISCUSSION TITLE
Allow server owners to accept donations

### DISCUSSION THREAD

**Discussion by ryanalexander** [stoatchat/discussions#77] [Feature Suggestions]
Incenticising moderation teams for large servers can sometimes be difficult, what I am proposing is a way (on platform) for servers to accept payment for features in the server.

This would allow revolt servers to  be a source of income and incentivise both large servers and content creators to switch over.

Revolt could take a small processing fee from the donation.

**Comment by oumellouk** [comment]
agree in this idea , and supports pateron or kofi will be better cuz servers owners loses much time to moderate servers and editing on it to make it happy and good place for users

**Comment by oleggtro** [comment]
I think Revolt shouldn't take a cut as its open source software. Revolt should be financed via direct donations.
Just use paypal for payment or sth
